lib/ae_mdw/contracts.ex

```diff
@@ -222,14 +222,12 @@ defmodule AeMdw.Contracts do
   end
 
   defp build_calls_pagination(
-         [fname: fname_prefix],
+         [fname: fname],
          state,
          {first_call_txi, last_call_txi},
          cursor
        ) do
-    key_boundary =
-      {{fname_prefix, first_call_txi, @min_idx},
-       {fname_prefix <> Util.max_256bit_bin(), last_call_txi, @max_idx}}
+    key_boundary = {{fname, first_call_txi, @min_idx}, {fname, last_call_txi, @max_idx}}
```

Comment:
Don't know if we can solve this way because in the docs and historically it works with `fname` prefix. 
Could you share how it would affect or not the client apps and how would be the policy for the general public?

Reply:
I am not sure, since 1.0 it works as a prefix:
https://github.com/aeternity/ae_mdw/blob/a7175ed73078a378f50647417105acc786c5191b/lib/ae_mdw_web/controllers/contract_controller.ex#L279
Described also in the [README.md](https://github.com/aeternity/ae_mdw/blob/1.0.9/README.md?plain=1#L3151)

Reply:
Here is an example from 1.0 for all Oracle.* internal calls:
https://github.com/aeternity/ae_mdw/blob/a7175ed73078a378f50647417105acc786c5191b/README.md?plain=1#L3189

Reply:
@jyeshe you're right 100%, what would you suggest we do? We can't return invalid results

Reply:
I am sure you can come with an intelligent solution. That would be a very interesting problem. I would start looking at secondary indexes implementation or negotiate with @thepiwo if it would be acceptable for public and internal client apps.